coordinator/core/marbleapi_test.go

```diff
@@ -545,14 +549,12 @@ func (ms *marbleSpawner) shortMarbleActivation(marbleType string, infraName stri
 	ms.assert.NoError(err)
 	marble = coreServerManifest.Marbles[marbleType]
 	// Validate Files
-	if marble.Parameters != nil {
-		if marble.Parameters.Files != nil {
-			ms.assert.Equal(marble.Parameters.Files, params.Files)
-		}
-		// Validate Argv
-		if marble.Parameters.Argv != nil {
-			ms.assert.Equal(marble.Parameters.Argv, params.Argv)
-		}
+	if marble.Parameters.Files != nil {
```

Comment:
Is the check for `nil` obsolete?

Reply:
Good catch, the nil check is obsolete, but we need to compare the file entries individually instead of comparing maps

Reply:
Ah, so gRPC is making sure a request can not have an empty `Parameters` field?

Reply:
No, this is a problem with the testing framework.
We compare `map[string]string` to `map[string][]bytes`, which always returns false.
We fix this by comparing the map values one by one instead all at the same time, since comparing `string` and `[]byte` evaluates as true, if the values are the same